torchgeo/datamodules/ssl4eo_benchmark_landsat.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""SSL4EO datamodule."""
+
+from typing import Any
+
+import kornia.augmentation as K
+
+from ..datasets import SSL4EOLBenchmark
+from ..transforms import AugmentationSequential
+from .geo import NonGeoDataModule
+
+
+class SSL4EOLBenchmarkDataModule(NonGeoDataModule):
+    """LightningDataModule implementation for the SSL4EO-L dataset.
+
+    .. versionadded:: 0.5
+    """
+
+    mean = 0
+    std = 255
```

Comment:
I believe these need to be tensors for older versions of Kornia (I fixed this in newer versions). Anyway, these are already the defaults in the base class, so you don't have to change them